src/Kunstmaan/TranslatorBundle/Controller/TranslatorCommandController.php

```diff
@@ -74,6 +74,8 @@ public function importForcedAction()
      */
     public function exportAction()
     {
+        // NEXT_MAJOR this route will be removed because we are now using the common export logic
+        @trigger_error('Calling this route is deprecated since KunstmaanNodeBundle 5.1 and will be removed in KunstmaanNodeBundle 6.0. Use the new route in TranslatorController that uses common export logic.', E_USER_DEPRECATED);
```

Comment:
```suggestion
        @trigger_error('Using the "KunstmaanTranslatorBundle_command_export" route is deprecated since KunstmaanTranslatorBundle 5.4 and will be removed in KunstmaanTranslatorBundle 6.0. Use the new route in TranslatorController that uses common export logic.', E_USER_DEPRECATED);
```


I don't understand the `Use the new route in TranslatorController that uses common export logic.` part, is there a replacement action? Or is it just the default adminlist export you mean? Otherwise I would reword this to `Use the default adminlist bundle export functionality instead`